frontend/src/components/Ubication/Ubication.module.css

```
@@ -0,0 +1,16 @@
+.ubication {
+  width: 250px;
+  background-color: #ffe086;
```

Sugerencia: 
seria bueno en los proximos componentes agregar las variables 👌
```suggestion
  background-color: var(--bg-color);
```